src/rrdp-squeezed/rrdp_squeezed.ml

```diff
@@ -42,7 +42,8 @@ end
 let get_running_domains xc =
 	Xenctrl.domain_getinfolist xc 0 |> List.map (fun di -> di.Xenctrl.domid)
 
-open Xenstore_watch
+module D=Debug.Make(struct let name="rrdd-plugins" end)
+include Ez_xenstore_watch.Make(D)
```

Comment:
I think in the Xenopsd PR made sense to include the module. Here I'd have probably just `open`ed it. I don't think that this will make much difference in the end, so feel free to keep it as it is.

Reply:
You can's say `open X(Y)` and hence it would look like this:

```
module XSW = Ez_xenstore_watch.Make(D)
open XSW
```